components/raftstore-v2/tests/integrations/mod.rs

```diff
@@ -50,19 +52,19 @@ mod test_read;
 mod test_status;
 
 #[derive(Clone)]
-struct TestRouter(StoreRouter<KvTestEngine, RaftTestEngine>);
+struct TestRouter(ServerRaftStoreRouter<KvTestEngine, RaftTestEngine>);
 
 impl Deref for TestRouter {
     type Target = StoreRouter<KvTestEngine, RaftTestEngine>;
 
     fn deref(&self) -> &Self::Target {
-        &self.0
+        self.0.router()
     }
 }
 
 impl DerefMut for TestRouter {
     fn deref_mut(&mut self) -> &mut Self::Target {
-        &mut self.0
+        self.0.router_mut()
```

Comment:
Should make target as `ServerRaftStoreRouter` instead of `StoreRouter`. raftstore should only export one type of router.

Reply:
Good suggestion.